lib/Remotery.c

```diff
@@ -4176,12 +4192,12 @@ static rmtError Remotery_Constructor(Remotery* rmt)
     assert(rmt != NULL);
 
     // Set default state
-    rmt->server = NULL;
+
+    // set all pointers to NULL, incl. rmt->opengl etc,
+    // so they're initialized even if we error out early
+    memset(rmt, 0, sizeof(*rmt));
+
```

Comment:
I know it's a pain but could you just copy the null pointers set below that up to here and rewarp them with their defines?

I don't want to set a precedent for using memset to set default state on an object in its constructor.